app/plugins/cron.js

```diff
@@ -0,0 +1,14 @@
+const cron = require('node-cron')
+const processComplianceApplications = require('../messaging/application/process-compliance-applications')
+const { complianceScheduler } = require('../config')
+
+module.exports = {
+  plugin: {
+    name: 'processComplianceApplications',
+    register: async () => {
+      cron.schedule(complianceScheduler, async () => {
```

Comment:
Based on discussions on Teams, it looks like if we are keeping the cron within this service we may need to look at the Helm charts to make sure the pod cannot scale beyond one to avoid the cron running in multiple places with side affects

Reply:
Could we achieve the same thing using Azure Timer Trigger Function?

https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-timer?tabs=in-process&pivots=programming-language-csharp